py2many/ast_helpers.py

```diff
@@ -23,7 +23,7 @@ def create_ast_node(code, at_node=None):
 
 def create_ast_block(body, at_node=None):
```

Comment:
this should be declared as `-> ast.If` (or `ASTxRewrittenIf`)?  Then its uses do not need a `cast`